.github/workflows/testing.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           rm yarn.lock
           git grep -lz '@solarwinds-apm/' | xargs -0 sed -ie 's:@solarwinds-apm/:@${{ github.repository_owner }}/solarwinds-apm-:g'
           git grep -lz -G 'solarwinds-apm[^-]' | xargs -0 sed -rie 's:solarwinds-apm([^-]):@${{ github.repository_owner }}/solarwinds-apm\1:g'
+          yarn install
```

Comment:
Workflow was currently crashing when not running this before any other yarn command